connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -320,6 +320,18 @@ public void putTaskConfigs(final @PathParam("connector") String connector,
         completeOrForwardRequest(cb, "/connectors/" + connector + "/tasks", "POST", headers, taskConfigs, forward);
     }
 
+    @PUT
+    @Path("/{connector}/fence")
+    public Response fenceZombies(final @PathParam("connector") String connector,
+                             final @Context HttpHeaders headers,
+                             final @QueryParam("forward") Boolean forward,
+                             final byte[] requestBody) throws Throwable {
+        FutureCallback<Void> cb = new FutureCallback<>();
+        herder.fenceZombies(connector, cb, InternalRequestSignature.fromHeaders(requestBody, headers));
+        completeOrForwardRequest(cb, "/connectors/" + connector + "/fence", "PUT", headers, requestBody, forward);
+        return Response.ok().build();
```

Comment:
This is to force a 200 OK response instead of a 204 no content response, which would be returned otherwise. I'd just use a 204 except the KIP specifies that this endpoint should "serve an empty-bodied 200 response" and I wanted to stick to that.

Given that this endpoint is internal and it's a tiny detail, I'd be fine with switch to a 204 response if it's alright with you.

Reply:
As far as I can tell the other internal endpoint returns 204 so I'd be in favor of doing the same here

Reply:
👍 done